zeebe-benchmark/profiling.yaml

```diff
@@ -0,0 +1,23 @@
+camunda-platform:
```

Comment:
I guess I would move it out of the chart directory since it is an additional values file that **can** be used together.

If we start to package the helm chart this will not be really visible I guess, idk